builder/scaleway/config.go

```diff
@@ -127,7 +127,7 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 	}
 
 	if c.BootType == "" {
-		c.BootType = "bootscript"
+		c.BootType = "local"
```

Comment:
Hi there, please update the documentation to reflect the new default https://github.com/hashicorp/packer/pull/9853/files#diff-e77a0a57d71e4aefbf0072f82a6072b2R65. Once changed you can run `make generate` to update the website doc files.

Reply:
done